api.web.yaml

```diff
@@ -1132,6 +1132,39 @@ components:
           type: string
           description: Validation error message explaining why the request failed
 
+    ProjectBase:
+      type: object
+      description: Base project properties
+      required:
+        - title
+      properties:
+        title:
+          type: string
+          description: Title of the project
+        description:
+          type: string
+          description: Description of the project
+
+    Project:
```

Comment:
I think we should be careful here about the specialisation here . I don't think we should ever respond with a bare project with no environment/service type (please disregard if that's not what you were thinking @jhughes982!) – happy to hear thoughts though